packages/trpc/src/concentrated-liquidity.ts

```diff
@@ -106,34 +106,42 @@ export const concentratedLiquidityRouter = createTRPCRouter({
   getLiquidityPerTickRange: publicProcedure
     .input(z.object({ poolId: z.string() }))
     .query(({ ctx, input }) => getLiquidityPerTickRange({ ...ctx, ...input })),
-  getBaseTokens: publicProcedure.query(({ ctx }) =>
-    getAssets({
-      assetLists: ctx.assetLists,
-      onlyVerified: true,
-    })
-      .map((asset) => {
-        const assetListAsset = getAssetFromAssetList({
-          assetLists: ctx.assetLists,
-          coinMinimalDenom: asset.coinMinimalDenom,
-        });
+  getBaseTokens: publicProcedure
+    .input(
+      z.object({
+        onlyVerified: z.boolean().default(false),
+        includePreview: z.boolean().default(false),
+      })
+    )
+    .query(({ ctx, input }) =>
+      getAssets({
+        assetLists: ctx.assetLists,
+        onlyVerified: input.onlyVerified,
+        includePreview: input.includePreview,
+      })
+        .map((asset) => {
+          const assetListAsset = getAssetFromAssetList({
+            assetLists: ctx.assetLists,
+            coinMinimalDenom: asset.coinMinimalDenom,
+          });
 
-        if (!assetListAsset) return;
+          if (!assetListAsset) return;
 
-        return {
-          chainName: assetListAsset.rawAsset.chainName,
-          token: new CoinPretty(
-            {
-              coinDenom: asset.coinDenom,
-              coinDecimals: asset.coinDecimals,
-              coinMinimalDenom: asset.coinMinimalDenom,
-              coinImageUrl: asset.coinImageUrl,
-            },
-            0
-          ).currency,
-        };
-      })
-      .filter((asset): asset is NonNullable<typeof asset> => Boolean(asset))
-  ),
+          return {
+            chainName: assetListAsset.rawAsset.chainName,
+            token: new CoinPretty(
```

Comment:
Why not just return the currency object directly instead of use CoinPretty?

Reply:
Ha ha, that's right. I'm not sure what happened there. I'll push a commit